scrapi/migrations.py

```diff
@@ -0,0 +1,36 @@
+import logging
+# import functools
+
+from scrapi import tasks
+from scrapi import settings
+from scrapi.linter import RawDocument
+from scrapi.processing.elasticsearch import es
+from celery import Celery
+
+app = Celery()
+app.config_from_object(settings)
+
+logger = logging.getLogger()
+
+
+@tasks.task_autoretry(default_retry_delay=30, max_retries=5)
+def rename(doc, source=None, target=None, dry=True):
+    assert source and target, "To run this migration you need both a source and a target"
+    assert source != target, "Can't rename {} to {}, names are the same".format(source, target)
+
+    raw = RawDocument({
```

Comment:
Rather than renormalizing it, we may be able to do something like
```
doc_attrs = dict(doc)
doc_attrs['source'] = target
DocumentModel.create(**attrs).save()
```